packages/terra-avatar/src/variants/Avatar.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import memoize from 'memoize-one';
+import 'terra-base/lib/baseStyles';
+import Utils from '../common/AvatarUtils';
+import styles from '../common/Avatar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Specifies the alternative text for the image.
+   */
+  alt: PropTypes.string.isRequired,
+  /**
+   * Sets the background color. Defaults to `auto`. Accepted color variants are theme specific.
+   * One of: `'auto'`, `'neutral'`, `'one'`, `'two'`, `'three'`, `'four'`, `'five'`, `'six'`, `'seven'`, `'eight'`, `'nine'`, `'ten'`.
+   */
+  color: PropTypes.oneOf(['auto', 'neutral', 'one', 'two', 'three', 'four',
+    'five', 'six', 'seven', 'eight', 'nine', 'ten']),
+  /**
+   * Value used for the hash function when color is set to `auto`. If not provided, hash function utilizes alt.
+   */
+  hashValue: PropTypes.string,
+  /**
+   * The image to display.
+   */
+  image: PropTypes.string,
+  /**
+   * One or two letters to display.
+   */
+  initials: PropTypes.string,
+  /**
+   * Whether to hide avatar from the accessiblity tree.
+   */
+  isAriaHidden: PropTypes.bool,
+  /**
+   * Whether the person is deceased. Overrides any color variant.
+   */
+  isDeceased: PropTypes.bool,
+  /**
+   * Overrides the default size.
+   */
+  size: PropTypes.string,
```

Comment:
refactored `width` and `height` props to just a `size` prop. cc @neilpfeiffer

Reply:
@yuderekyu Would it make for sense to call this fontSize? I would think size would directly affect the size of the component. Also, I think this could use more context in the description to indicate this is for scaling the initials.

Reply:
Ah good deal then!